packages/utils/upgrade/src/modules/upgrader/upgrader.ts

```diff
@@ -190,7 +240,7 @@ export class Upgrader implements UpgraderInterface {
     const updatedPackageJSON = json.root();
 
     if (this.isDry) {
-      this.logger?.debug(`Skipping dependencies update (${chalk.italic('dry mode')}`);
+      this.logger?.debug(`Skipping dependencies update (${chalk.italic('dry mode')})`);
```

Comment:
Side note (not to address here, you didn't change anything) but we should standardize printing dry mode messages instead of appending like this